Sources/Gui/Main.cpp

```diff
@@ -57,13 +57,23 @@
 #endif
 
 
+#ifdef _MSC_VER>=1900	// Visual Studio 2015 or higher
+extern "C" { FILE __iob_func[3] = { *stdin,*stdout,*stderr }; }
```

Comment:
I don't think this works, because [`__iob_func` should be defined as a function](https://groups.google.com/forum/#!topic/dislin-users/kagVl9LVFJA) returning the array of `FILE`s.